skycultures/indian/index.json

```diff
@@ -728,98 +728,99 @@
-    "HIP 24608": [{"pronounce": "Bramha-hṛdaya", "english": "Heart of the Creator"}],
-    "HIP 25336": [{"pronounce": "Kārtikeya", "english": "Son of lord Shiva"}],
-    "HIP 25428": [{"pronounce": "Agni", "english": "Fire"}],
-    "HIP 25930": [{"pronounce": "Chitralekhā", "english": "As beautiful as a picture"}],
-    "HIP 26311": [{"pronounce": "Aniruddha", "english": "Unbounded"}],
-    "HIP 26727": [{"pronounce": "Uṣā", "english": "Dawn"}],
-    "HIP 27366": [{"pronounce": "Kārtavīrya"}],
-    "HIP 27989": [{"pronounce": "Ārdrā", "english": "Moist"}],
-    "HIP 30438": [{"pronounce": "Agastya", "english": "A great sage who went south"}],
-    "HIP 31681": [{"pronounce": "Anala", "english": "Fire"}],
-    "HIP 32349": [{"pronounce": "Lubdhaka", "english": "Hunter"}],
-    "HIP 36850": [{"pronounce": "Aditi", "english": "Mother of gods"}],
-    "HIP 37826": [{"pronounce": "Diti", "english": "Mother of demons"}],
-    "HIP 49669": [{"pronounce": "Maghā", "english": "Bountiful one"}],
-    "HIP 53910": [{"pronounce": "Pulaha", "english": "One of the seven sages"}],
-    "HIP 54061": [{"pronounce": "Kratu", "english": "One of the seven sages"}],
-    "HIP 58001": [{"pronounce": "Pulasta", "english": "One of the seven sages"}],
-    "HIP 59774": [{"pronounce": "Atri", "english": "One of the seven sages"}],
-    "HIP 62956": [{"pronounce": "Angirā", "english": "One of the seven sages"}],
-    "HIP 65378": [{"pronounce": "Vaśisṭha", "english": "One of the seven sages"}],
-    "HIP 65474": [{"pronounce": "Chitrā", "english": "Spectacular (Spica)"}],
-    "HIP 65477": [{"pronounce": "Arundhati", "english": "Wife of Vaśisṭha"}],
-    "HIP 67301": [{"pronounce": "Mārīchi", "english": "One of the seven sages"}],
-    "HIP 69673": [{"pronounce": "Svāti", "english": "Honest one"}],
-    "HIP 80763": [{"pronounce": "jyeṣṭhā", "english": "Eldest"}],
-    "HIP 91262": [{"pronounce": "Abhijit", "english": "Invincible"}],
-    "HIP 97649": [{"pronounce": "Śravaṇa", "english": "To hear or Ear"}],
-    "HIP 110395": [{"pronounce": "Śatabhiṣak"}],
-    "NAME Jupiter": [{"english": "Mentor", "pronounce": "Guru", "translators_comments": "Native name for Jupiter"}],
-    "NAME Mars": [{"english": "Auspicious one", "pronounce": "Maṇgala", "translators_comments": "Native name for Mars"}],
-    "NAME Mercury": [{"english": "Wise one", "pronounce": "Budha", "translators_comments": "Native name for Mercury"}],
-    "NAME Moon": [{"english": "Delighter", "pronounce": "Candra", "translators_comments": "Native name for the Moon"}],
-    "NAME Saturn": [{"english": "Slow mover", "pronounce": "Śani", "translators_comments": "Native name for Saturn"}],
-    "NAME Sun": [{"english": "Vigorous", "pronounce": "Sūrya", "translators_comments": "Native name for the Sun"}],
-    "NAME Venus": [{"english": "Bright one", "pronounce": "Śukra", "translators_comments": "Native name for Venus"}]
+    "HIP 746": [{"native": "जमदग्निः", "pronounce": "Jamadagni", "english": "Jamadagni"}],
```

Comment:
The old version had an explanatory addition. Was it wrong? If possible, proper names (from mythology?) should be explained a bit. Of course, the main text in description.md could also be improved, and such explanations could also be listed in a table there (agreed, less screen clutter...). But now it seems interested non-native users just have less information.